content/applications/websites/website/web_design/building_blocks.rst

```diff
@@ -0,0 +1,133 @@
+block, and drag and drop it to the correct zone of your page. You can add as many blocks as needed.
+
+To edit the content of a building block, click on it and go to the :guilabel:`Customize` tab.
+
+.. Note::
+   The features available in the :guilabel:`Customize` tab depend on the block you selected.
+
+Color preset and background
+===========================
+
+You can customize and apply :guilabel:`Color Presets` to your site's building block. To proceed,
+select a building block, go to the :guilabel:`Customize` tab, click the :guilabel:`Background`
+button, and select the desired :guilabel:`Preset`.
+
+When you modify a color preset that has been previously applied to specific elements on your site,
+those elements automatically update to reflect the changes made to the selected preset.
+
+.. seealso::
+   :doc:`Website themes <themes>`
+
+Layout: Grid vs Cols
+====================
+
+Go to the :guilabel:`Customize` tab and then change the :guilabel:`Layout` of a selected element to
+:guilabel:`Grid` or :guilabel:`Cols` in the :guilabel:`Banner` section.
+
+Grid
+----
+
+The :guilabel:`Grid` layout allows you to reposition and resize elements, such as images or text, by
+dragging and dropping them to the desired place.
+
+.. image:: building_blocks/grid-layout.png
+   :alt: When the grid layout is selected, choose an image and drag and drop it where needed.
+
+.. tip::
+   Position images behind the text by using the above/below icons.
+
+   .. image:: building_blocks/superimpose-images-to-text.png
+      :alt: Use the above/below icons to superimpose images to text.
+
+Cols
+----
+
+The :guilabel:`Cols` feature is available for some blocks like :guilabel:`Columns` or
+:guilabel:`Image - Text` blocks. By choosing the :guilabel:`Cols` :guilabel:`Layout`, you can
+determine the number of elements per line within the block.
+
+.. image:: building_blocks/elements-per-line.png
+   :alt: Set the number of elements per line in the block.
+
+.. note::
+   In the mobile view, only one element per column is used by default for better visibility.
+   However, you can adjust this setting without impacting the desktop view.
+
+Duplicating a building block
+============================
+
+You can replicate a building block by clicking on the duplicate icon. Once duplicated, the new block
+appears on your website beneath the original one.
+
+.. image:: building_blocks/duplicate-container.png
+   :alt: Replicating a building block
+
+Re-ordering building blocks
+===========================
+
+To re-order the building blocks on your site, select the block you want to move and click the up
+arrow to move it before the previous section or click the down arrow to move it after the following
+section.
+
+You can also use the drag-and-drop icon to move your block manually.
+
+.. image:: building_blocks/re-ordering-blocks.png
+   :alt: Re-order building blocks by using the up and down arrow icons.
+
+Show/Hide on mobile
```

Comment:
```suggestion
Visibility of building blocks per device type or condition
```
I would change the heading to speak about all visibility options.

Reply:
I do not really like the title you suggested; it's too long for me. If you can live with it, I would prefer to keep the title I wrote.

Reply:
or maybe to speak about "visibility on mobile"?

Reply:
updated as discussed live